### ISSUE TITLE
KEGG 'EFFICACY' keyword without special parsing.

### ISSUE THREAD

**Issue by nc1m** [cokelaer/bioservices#218]
Hi,

the following warning appeared while using the library:

```
WARNING [bioservices:keggparser:119]:  Found keyword EFFICACY, which has not special
    parsing for now. please report this issue with the KEGG 
    identifier (D00002                      Drug) into github.com/bioservices. Thanks T.C.

```

**Comment by cokelaer** [comment]
@nc1m Thanks for reporting this warning and using bioservices !
I have included efficacy in the code. Most probably you had no issue because efficacy is just a string but in the new version that I will release in a couple of days this warning will be removed. stay tuned

**Comment by cokelaer** [comment]
Fixes in v1.10 available on pypi
